### ISSUE TITLE
'Only autoplay videos on WiFi' setting

### ISSUE THREAD

**Issue by jariz** [jariz/Dumpert#60]
Title speaks for itself.

**Comment by CytoDev** [comment]
Would require a rewrite of ViewItemActivity (or ViewItemFragment, depending on branch) since the click action registers as a web intent.

**Comment by jariz** [comment]
How so? I'm talking about automatically playing a video which is something we already have a setting for ("Play videos automatically" in data settings), it just disables it across the application entirely.
All that needs to be done is have that same behavior but only if you're on WiFi.

**Comment by CytoDev** [comment]
Yes, auto-play is built in. But try to disable that preference and open a video. You will be linked to the video in a web intent instead of the video player starting. At least on all builds I tested.

**Comment by jariz** [comment]
I agree that it would be better that rather than a web intent, it uses the build in player like it does when autoplay is disabled, but that's an issue for another ticket for now.
